src/Commands/SyncCommands.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace Drupal\opening_hours\Commands;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\opening_hours\Services\SyncService;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Sync drush command.
+ */
+class SyncCommands extends DrushCommands {
+
+  /**
+   * Sync service.
+   *
+   * @var \Drupal\opening_hours\Services\SyncService
+   */
+  private $syncService;
```

Comment:
Are you sure that this class will never be extended?
Otherwise I'd go for protected.

Reply:
This drush command class should never be extended (you end up with 2 classes with the same commands).

Reply:
I will make this class final.